OJP/OJP_JourneySupport.xsd

```diff
@@ -644,7 +644,7 @@
 			<xs:choice minOccurs="0">
 				<xs:element name="NumberOfResults" type="xs:positiveInteger" minOccurs="0">
 					<xs:annotation>
-						<xs:documentation>The number of trip results that the user wants to see at least (TRIP REQUEST POLICY.maximumNumberofJourneys). Be aware that some implementations will deliver one of the TripResults before the indicated departure time. This means one always must check the results against the parameters before using them.</xs:documentation>
+						<xs:documentation>The number of trip results that the user wants to see at least (according to the Transmodel TRIP REQUEST POLICY.maximumNumberofJourneys). Be aware that some implementations will deliver one of the TripResults before the indicated departure time. This means that you can't assume that you get the exact number of results that you indicated in the request from the server.</xs:documentation>
```

Comment:
Yes, this makes it clear where the maximumNumberofJourneys comes from. BUT it still sounds buggy - "least" number of trip results would match *minimumNumberofJourneys* (if this exists), not the current *maximumNumber...*.

Reply:
@ue71603 @sgrossberndt For me there remains just this as an open point, otherwise everything fine.

Reply:
@trurlurl Did you see the latest correction of @ue71603 ?
https://github.com/VDVde/OJP/pull/443/commits/32b1381f02e0d443a627d86a95cbceac3b61d3e4

Reply:
> @trurlurl Did you see the latest correction of @ue71603 ? [32b1381](https://github.com/VDVde/OJP/commit/32b1381f02e0d443a627d86a95cbceac3b61d3e4)

@sgrossberndt Yes, I've seen the change resolving the initial question. The remaining problem is just that _**maximum**Number_ suggests a "**at most**" while the text says "the number ... the user wants to see **at least**". In Transmodel I've found _MaximumNumberOfResults_, but no corresponding _MinimumNumberOfResults_. I'm forwarding the suggestion of simply removing the reference to Transmodel.